src/coreclr/src/debug/ee/wks/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ if (CLR_CMAKE_TARGET_WIN32)
 
     preprocess_compile_asm(ASM_FILES ${ASM_FILE} OUTPUT_OBJECTS ASM_OBJECTS)
 
-    add_library_clr(cordbee_wks OBJECT ${CORDBEE_SOURCES_WKS} ${ASM_OBJECTS})
+    add_library_clr(cordbee_wks STATIC ${CORDBEE_SOURCES_WKS} ${ASM_OBJECTS})
```

Comment:
Why is this needed? It seems unfortunate that we need to separate them from the coreclr static library.

Reply:
These targets preprocess and compile ASM files and our put result in output, but not where cmake expects obj files to be in an object library (it looks in an intermediate folder that itself creates).  
 It may even be a bug in cmake, i am not sure, but otherwise this does not build on arm/arm64.